roles/podman/tasks/main.yml

```diff
@@ -0,0 +1,83 @@
+---
+- name: Allocate subuids for {{ podman_user }} user
+  subuid:
```

Comment:
It turns out, this (and subgid) are modules in the installer.

Reply:
yes... chat with @shanemcd and validated that in recent RHEL8 and RHEL9 its no longer needed and what the role configured is now default